hollow/src/main/java/com/netflix/hollow/api/producer/ListenerSupport.java

```diff
@@ -32,13 +32,17 @@
 import java.util.Map;
 import java.util.Set;
 import java.util.concurrent.CopyOnWriteArraySet;
+import java.util.logging.Logger;
 
 /**
  * Beta API subject to change.
  *
  * @author Tim Taylor {@literal<tim@toolbear.io>}
  */
 final class ListenerSupport {
+
+    private final Logger log = Logger.getLogger(ListenerSupport.class.getName());
```

Comment:
Minor but can we make this static final?

`private static final Logger LOG = Logger.getLogger(ListenerSupport.class.getName());`

Similar to the logger in HollowClientUpdater.